CONTRIBUTING.adoc

```diff
@@ -0,0 +1,189 @@
+:hide-uri-scheme:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+:toc:
+
+:rnrh-test-folder: https://github.com/native-html/core/tree/master/packages/render-html/src/__tests__
```

Comment:
Fix this link; there may be others that I missed
I suggest searching for `native-html/core` and `render-html` in the codebase